spec/services/doc_auth/mock/result_response_spec.rb

```diff
@@ -648,16 +651,60 @@
   end
 
   context 'when a selfie check is performed' do
-    let(:input) { DocAuthImageFixtures.document_front_image }
-    let(:selfie_check_performed) { true }
-
-    it { expect(response.selfie_check_performed?).to eq(true) }
+    describe 'and it is successful' do
+      let(:input) do
+        <<~YAML
+          portrait_match_results:
+            FaceMatchResult: Pass
+            FaceErrorMessage: 'Successful. Liveness: Live'
+          doc_auth_result: Passed
+          failed_alerts: []
+        YAML
+      end
+      let(:selfie_check_performed) { true }
+
+      it 'returns the expected values' do
+        selfie_results = {
+          FaceMatchResult: 'Pass',
+          FaceErrorMessage: 'Successful. Liveness: Live',
+        }
+
+        expect(response.selfie_check_performed?).to eq(true)
+        expect(response.extra[:portrait_match_results]).to eq(selfie_results)
+      end
+    end
+
+    describe 'and it is not successful' do
+      let(:input) do
+        <<~YAML
+          portrait_match_results:
+            FaceMatchResult: Fail
+            FaceErrorMessage: 'Successful. Liveness: Live'
```

Comment:
Should we use an error message?

Reply:
Can you tell me more about what you mean? I think the front end messages will be handled starting with [LG-11893](https://cm-jira.usa.gov/browse/LG-11893).

For now I thought this ticket was about having the mock proofer return those messages as expected. Perhaps I'm missing something though!

Reply:
I think I misunderstood what you meant. I talked with Charley and I think he had an idea of what you meant - that "because  `FaceMatchResult: Fail` is failing, we should do something other than `FaceErrorMessage: 'Successful. Liveness: Live'`".

I agree that it is confusing, but I think it is a valid state to get back [based on my reading of page 228 here](https://drive.google.com/file/d/1MmXMYkFqPsn2VMbsMVYAUyC-amLRHVCp/view?usp=sharing).

- `FaceErrorMessage` returns the result of the liveness check
- `FaceMatchResult` returns the result of the portrait match check, which won't happen unless the liveness check succeeds

So you can have a passing liveness check / `FaceErrorMessage` that fails the portrait match / `FaceMatchResult`.